sympy/solvers/tests/test_solvers.py

```diff
@@ -329,10 +329,10 @@ def test_quintics_2():
         CRootOf(x**5 - 6*x**3 - 6*x**2 + x - 6, 3),
         CRootOf(x**5 - 6*x**3 - 6*x**2 + x - 6, 4)]
 
-
+@slow
 def test_quintics_3():
     y = x**5 + x**3 - 2**Rational(1, 3)
-    assert solve(y) == solve(-y) == []
+    assert solve(y) == solve(-y)
```

Comment:
Presumably the change to `real_roots` affects something in `solve`. We need to consider whether that change is wanted. It might be better to disable that change for now by using `real_roots(..., extension=False)` or something.

Reply:
Or otherwise this test should be updated to show what is actually returned.

Reply:
I agree with checking what is actually returned. I will add this in.

Reply:
I'm wondering where it is in solve that something is effected by the change in this PR and whether is is a good thing that it is affected. I assume that `all_roots` is being used somewhere but it might be better to use extension=False or something.

Reply:
Yup `solve()` calls `poly.all_roots()`. I'll get back to this test after how `real_roots()` and `all_roots()` are called is finalized

Reply:
I've changed this test back as extension=False is default, so solve indeed returns an empty list.